maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractMapVersionsPhase.java

```diff
@@ -118,6 +125,15 @@ public ReleaseResult execute(
             throws ReleaseExecutionException {
         ReleaseResult result = new ReleaseResult();
 
+        List<String> additionalExcludes = releaseDescriptor.getCheckModificationExcludes();
+
+        if (additionalExcludes != null) {
+            for (String additionalExclude : additionalExcludes) {
+                exclusionPatterns.add(
+                        additionalExclude.replace("\\", File.separator).replace("/", File.separator));
```

Comment:
File.separator is platform dependent, but I'm not sure it should be here. That is, exclusions should be specified in a platform independent, standard way. I could be wrong about that but if I am, please elaborate in detail on how this works with references to docs.

Reply:
I fully referenced the implementation in `ScmCheckModificationsPhase`.
There is a comment explaining `// SelectorUtils expects File.separator, don't standardize`

Reply:
I'm having trouble finding that reference. Regardless, we shouldn't be using plexus for this any more,. Possibly maven-shared-utils but ideally not even that.